example/src/main/scala/example/HelloWorldAdvanced.scala

```diff
@@ -24,22 +23,24 @@ object HelloWorldAdvanced extends App {
   private val server =
     Server.port(PORT) ++              // Setup port
       Server.paranoidLeakDetection ++ // Paranoid leak detection (affects performance)
-      Server.app(fooBar ++ app)       // Setup the Http app
+      Server.app(fooBar ++ app) ++    // Setup the Http app
+      Server.auto                     // Specifying a Transport type (by default Auto)
```

Comment:
instead of `auto` we might be using `default` as we will be consistent across project